src/app/pages/wallets/my-wallets/wallet-form-modal/wallet-form-modal.component.spec.ts

```diff
@@ -33,4 +35,36 @@ describe('WalletFormModalComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('Should display error message when invalid data is provided', () => {
+    beforeEach(() => {
+      ERROR_MESSAGE_REQUIRED = 'Name is required'
+      ERROR_MESSAGE_MAXLENGTH = 'Name can\'t have more than 20 characters'
+    })
+
+    it('Empty name', () => {
```

Comment:
Those `it('something')` should make sense in english. "it empty name" does not make sense. I would go maybe with "it should show validation error if name input is empty". Don't be afraid of long phrases :)